src/python/bot/tasks/minimize_task.py

```diff
@@ -1047,19 +1047,22 @@ def do_js_minimization(test_function, get_temp_file, data, deadline, threads,
   data = do_line_minimization(test_function, get_temp_file, data, deadline,
                               threads, cleanup_interval, delete_temp_files)
 
-  tokenizers = [js_tokenizer.paren_tokenizer, js_tokenizer.comma_tokenizer]
-  for tokenizer in tokenizers:
-    current_minimizer = js_minimizer.JSMinimizer(
-        test_function,
-        max_threads=threads,
-        deadline=deadline,
-        cleanup_function=process_handler.cleanup_stale_processes,
-        single_thread_cleanup_interval=cleanup_interval,
-        get_temp_file=get_temp_file,
-        delete_temp_files=delete_temp_files,
-        tokenizer=tokenizer,
-        token_combiner=html_tokenizer.combine_tokens,
-        progress_report_function=functools.partial(logs.log))
+  tokenizer = AntlrTokenizer(JavaScriptLexer)
+
+  current_minimizer = js_minimizer.JSMinimizer(
+      test_function,
+      max_threads=threads,
+      deadline=deadline,
+      cleanup_function=process_handler.cleanup_stale_processes,
+      single_thread_cleanup_interval=cleanup_interval,
+      get_temp_file=get_temp_file,
+      delete_temp_files=delete_temp_files,
+      tokenizer=tokenizer.tokenize,
+      token_combiner=tokenizer.combine,
+      progress_report_function=functools.partial(logs.log))
+
+  # Some tokens can't be removed until other have, so do 2 passes
+  for _ in range(2):
```

Comment:
nit question: is two always enough? if there is a condition we need to meet, should it be a `while` loop instead?

Reply:
Two is the number of iterations it was doing before, so this will work as well as it used to. It might be interesting to keep track of tokens left after each run and stop when minimization runs aren't decreasing the tokens remaining. The problem with this is that we will guarantee that we are doing one more run than we need to. Thoughts? @mbarbella-chromium

Reply:
We don't really have a good way to define a condition to say the test case is "minimized enough" so I think the current approach is fine. Many cases really only need one pass, but there are a few common issues that two passes helps enough with to justify it. Going beyond that is probably overkill.